app/controllers/api/v1/members_controller.rb

```diff
@@ -0,0 +1,11 @@
+class Api::V1::BillsController < ApplicationController
+  def index
```

Comment:
this controller is called "members_controller" but is titled as BillsController in the header. This will need to be updated otherwise will clash with our current main